commit bd99e6e4616fddeba7b53333308a57fb327709a8
Author: Fabio M. Di Nitto <fdinitto@redhat.com>
Date:   Fri Mar 26 18:18:19 2010 +0100

    cman: use hash_cluster_id by default
    
    This change is not yet suitable for upstream as it breaks minor rolling upgrades.
    
    Signed-off-by: Fabio M. Di Nitto <fdinitto@redhat.com>

diff --git a/cman/daemon/cman-preconfig.c b/cman/daemon/cman-preconfig.c
index ef5964f..2352670 100644
--- a/cman/daemon/cman-preconfig.c
+++ b/cman/daemon/cman-preconfig.c
@@ -50,7 +50,7 @@ static char *mcast_name;
 static char *cluster_name;
 static char error_reason[1024] = { '\0' };
 static hdb_handle_t cluster_parent_handle;
-static int use_hashed_cluster_id = 0;
+static int use_hashed_cluster_id = 1;
 
 /*
  * Exports the interface for the service
